.github/workflows/dotnetcore.yml

```diff
@@ -0,0 +1,19 @@
+name: .NET Core
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 2.2.108
```

Comment:
Not sure about the dotnet version.
Maybe we should use the one that it's pinned in the global file.

Reply:
There is one version used in .travis.yml as well